### PR TITLE
Update scan package to 1.0.0

### PR DESCRIPTION
**Pull request by xtreme-conor-nosal**
Update package name and version
Remove manual store setup as new packages have default configuration

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ The following components are new in Tanzu Application Platform v0.4.0:
 
 The following components have been updated in Tanzu Application Platform v0.4.0:
 
+- Supply Chain Security Tools for VMware Tanzu
```
Comment (hectorj2f) [inline release-notes.md:23]: To be consistent `use Supply Chain Security Tools` as you removed `VMware Tanzu`

File: install-components.md [hunk 2]
```diff
@@ -1883,153 +1883,64 @@ To install Supply Chain Security Tools - Scan (Scan Controller):
...
+1. Install the package by running (omit the `--values-file` line if installing with default configuration):
 
     ```console
     tanzu package install scan-controller \
-      --package-name scanning.apps.tanzu.vmware.com \
-      --version 1.0.0-beta.2 \
+      --package-name scst-scan.apps.tanzu.vmware.com \
+      --version 1.0.0 \
       --namespace tap-install \
       --values-file scst-scan-controller-values.yaml
```
Comment (hectorj2f) [inline install-components.md:1913]: Isn't it clear to me ? Do we really need `scst-scan-controller-values.yaml` now ?

Comment (hectorj2f) [reply install-components.md:1913]: The users would copy and paste this command, so better suggest it to add a yaml file but leave the command example as simpler as possible.

**Comment by RichardJJG** [comment]
Remove draft status when this is ready for merging, please.

**Comment by jncd** [comment]
@xtreme-conor-nosal Hi Conor, is this ready for merging?

**Pull request merged**